src/appleseed.studio/python/module.cpp

```diff
@@ -68,7 +68,7 @@ Project* new_project()
 
 Project* open_project(const char* project_path)
```

Comment:
I think I agree, I also prefer the second option. Moreover, the user is free to ignore the return value and use `current_project()` to check if the project has been successfully loaded:

    studio.open_project(project_path)
    if studio.current_project() is None:
       throw Exception(error_msg)